skypy/galaxy/size.py

```diff
@@ -38,11 +41,8 @@ def angular_size(physical_size, redshift, cosmology):
     Examples
     --------
     >>> from astropy import units
-    >>> from astropy.cosmology import FlatLambdaCDM
-    >>> cosmology = FlatLambdaCDM(H0=67.04, Om0=0.3183, Ob0=0.047745)
-    >>> size = 10.0 * units.kpc
-    >>> angular_size(size, 1, cosmology)
-    <Quantity 5.85990062e-06 rad>
+    >>> from skypy.galaxy.size import angular_size
+    >>> r = angular_size(10*units.kpc, 1)
```

Comment:
I worry that without a full explanation of what the decorator does it is kind of hard to understand the default behaviour in this example when cosmology isn't given. I would suggest reverting to an explicit cosmology e.g. Planck15 and add information to the docstring explaining the behaviour when cosmology is not given.
```suggestion
    >>> from skypy.galaxy.size import angular_size
    >>> r = angular_size(10*units.kpc, 1, Planck15)
```

Reply:
can you also suggest importing Planck15 from astropy.cosmology please

Reply:
Updated suggestion